src/Chapter2.hs

```diff
@@ -336,7 +337,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b l | b - a < 0 = []
```

Comment:
I think subtraction is redundant here, and you can just move `a` to the right side of `a` :slightly_smiling_face: 
```suggestion
subList a b l | b < a = []
```